### PR TITLE
Localized properties hints, label of shema's field

### PR DESCRIPTION
**Pull request by dborovsky**
Localized properties hints, label of shema's field

### CODE REVIEW

**Review by SebastianStehle — CHANGES_REQUESTED** [review]
It goes into the right direction :)

File: backend/src/Squidex.Domain.Apps.Core.Model/Schemas/NamedElementPropertiesBase.cs [hunk 1]
```diff
@@ -5,12 +5,89 @@
...
+        public string? Label
+        {
+            get
+            {
+                if (DefaultValuesLabel == null)
+                {
+                    return string.Empty;
+                }
+
+                if (DefaultValuesLabel.TryGetValue(CultureInfo.CurrentUICulture.ToString(), out var current))
```
[the remark this answers is not in the document]
Comment (dborovsky) [reply backend/src/Squidex.Domain.Apps.Core.Model/Schemas/NamedElementPropertiesBase.cs:24]: What you mean "Perhaps we can move this code to LocalizedValue"?

[the remark this answers is not in the document]
Comment (SebastianStehle) [reply backend/src/Squidex.Domain.Apps.Core.Model/Schemas/NamedElementPropertiesBase.cs:24]: It is a common behavior, we can add a method to LocalizedValue class.

File: frontend/app/shared/services/schemas.service.ts [hunk 2]
```diff
@@ -319,10 +319,14 @@ export class NestedFieldDto extends FieldDto {
     }
 }
 
+type DefaultValue<T> = { [key: string]: T | null | undefined };
+
 export class SchemaPropertiesDto {
     constructor(
         public readonly label?: string,
```
[the remark this answers is not in the document]
Comment (dborovsky) [reply frontend/app/shared/services/schemas.service.ts:326]: Do you mean implement getter in class FieldPropertiesDto? 
`this.localizedLabel[currentLanguage] ` - how better get user's current language? I don't see any method for this. I only see implementation in profile-menu-component as `public language = this.uiOptions.get('more.culture');`

[the remark this answers is not in the document]
Comment (SebastianStehle) [reply frontend/app/shared/services/schemas.service.ts:326]: I think we need a pipe for that...so we can inject the uiOptions

**Pull request closed without merging**